bus-mapping/src/rpc.rs

```diff
@@ -253,9 +309,59 @@ mod rpc_tests {
                 }
             ]
         }"#;
-        assert!(
-            serde_json::from_str::<EIP1186ProofResponse>(TARGET_PROOF).unwrap()
-                == proof
-        );
+        let target_proof =
+            serde_json::from_str::<EIP1186CustomProofResponse>(TARGET_PROOF)
+                .unwrap();
+        assert_eq!(proof.balance, target_proof.balance);
+        assert_eq!(proof.codeHash, target_proof.codeHash);
+        assert_eq!(proof.nonce, target_proof.nonce);
+        assert_eq!(proof.storageHash, target_proof.storageHash);
+        assert_eq!(proof.storageProof, target_proof.storageProof);
+    }
+
+    fn get_provider() -> GethClient<Http> {
+        let transport = Http::new(Url::parse("http://localhost:8545").unwrap());
```

Comment:
We should use an env var here. But never hardcore the address directly no?

Reply:
I like your suggestion!